docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -60,8 +60,10 @@ cloud providers is difficult.
 {% capture prerequisites %}
 
 1. One or more machines running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+
-1. 1GB or more of RAM per machine (any less will leave little room for your
+1. 2GB or more of RAM per machine (any less will leave little room for your
```

Comment:
Why change these requirements?

Reply:
Cluster creation reproducibly fails with 1GB of RAM.

Reply:
@timothysc @jbeda Although I was sceptical towards bumping this initially; it might make sense to do
kube has grown in size indeed :/

Reply:
+1 from me.